Assets/MirageXR/ContentTypes/Audio/Editors/Mobile/AudioCaptionEdit.cs

```diff
@@ -0,0 +1,93 @@
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class AudioCaptionEdit : MonoBehaviour
+{
+    [SerializeField] private Button _btnBackPreview;
+    [SerializeField] private Button _btnDoneEditing;
+    [SerializeField] private TMP_InputField _captionEditText;
+
+    [SerializeField] private GameObject _panelMain;
+    [SerializeField] private GameObject _panelCaptionEdit;
+    [SerializeField] private GameObject _panelCaptionPreview;
+
+    [SerializeField] private AudioCaptionPreview audioCaptionPreview;
+    private string text = string.Empty;
+
+    // Start is called before the first frame update
+    private void Start()
+    {
+        // Ensure all objects are properly assigned in the Unity Inspector
+        if (_panelCaptionPreview == null )
+        {
+            Debug.LogError("_panelCaptionPreview is not assigned ");
+            return; // Early exit to prevent accessing null references
+        }if (_panelMain == null )
+        {
+            Debug.LogError("_panelMain  is not assigned ");
+            return; // Early exit to prevent accessing null references
+        }if (_btnBackPreview == null )
+        {
```

Comment:
```suggestion
        }
        if (_panelMain == null )
        {
            Debug.LogError("_panelMain  is not assigned ");
            return; // Early exit to prevent accessing null references
        }
        if (_btnBackPreview == null )
        {
```